src/Auth/EmailVerification/EmailVerificationController.php

```diff
@@ -0,0 +1,108 @@
+<?php
+namespace Nodes\Api\Auth\EmailVerification;
+
+use Illuminate\Routing\Controller as IlluminateController;
+use Illuminate\Support\Facades\Input;
+use Nodes\Api\Auth\Exceptions\MissingUserModelException;
+use Nodes\Api\Routing\Helpers as ApiHelpers;
+use Nodes\Exceptions\Exception;
+
+/**
+ * Class EmailVerificationController
+ *
+ * @package Nodes\Api\Auth\EmailVerification
+ */
+class EmailVerificationController extends IlluminateController
+{
+    use ApiHelpers;
+
+    /**
+     * Reset password model
+     *
+     * @var \Nodes\Api\Auth\EmailVerification\EmailVerificationRepository
+     */
+    protected $emailVerificationRepository;
+
+    /**
+     * EmailVerificationController constructor.
+     * @param EmailVerificationRepository $emailVerificationRepository
+     */
+    public function __construct(EmailVerificationRepository $emailVerificationRepository)
+    {
+        $this->emailVerificationRepository = $emailVerificationRepository;
+
+        // Share variables with all views
+        view()->share([
+            'title' => 'Email verification',
+            'senderName' => (config('nodes.api.email-verification.from.name') != 'Nodes') ? config('nodes.api.email-verification.from.name') : config('nodes.project.name')
+        ]);
+    }
+
+    /**
+     * @param $token
+     * @param $email
+     * @return $this|\Illuminate\Contracts\View\Factory|\Illuminate\Http\RedirectResponse|\Illuminate\View\View
+     * @author Paulius Navickas <pana@nodes.dk>
+     */
+    public function index($token, $email)
+    {
+        // Validate token
+        $verificationToken = $this->emailVerificationRepository->getByUnexpiredToken($token);
```

Comment:
Since your field in your model is called `verification` instead of `token` this should be renamed to `getByUnexpiredVerification`.